test/test_services_constructor.jl

```diff
@@ -969,13 +970,12 @@ end
             resolution = Hour(1),
             optimizer = HiGHS_optimizer,
             store_variable_names = true,
-            optimizer_solve_log_print = true,
         )
     @test build!(
         ps_model;
         console_level = Logging.AboveMaxLevel,  # Ignore expected errors.
         output_dir = mktempdir(; cleanup = true),
-    ) == PSI.ModelBuildStatus.FAILED
+    ) == PSI.ModelBuildStatus.BUILT
```

Comment:
This test still concerns me. I thought we decided that this test should still fail because we have a double circuit where the flow is represented by a single variable, and the user has indicated conflicting directions.